src/ucp/rma/basic_rma.c

```diff
@@ -25,18 +25,81 @@
         return UCS_ERR_INVALID_PARAM; \
     }
 
+#define _USE_ZCOPY 1
```

Comment:
need to make this runtime option by using zero copy threshold vairable in ucp_context